paywall/src/config.js

```diff
@@ -89,7 +89,6 @@ export default function configure(
   }
   let supportedProviders = []
   let blockTime = 8000 // in mseconds.
-  const httpProvider = runtimeConfig.httpProvider
```

Comment:
this can be rebased out once the fix is in, I just wanted to get the relevant code reviewed.